resources/css/site.css

```diff
@@ -603,3 +603,49 @@ a.text-center.btn.btn-outline-light.btn__book {
 .card-img-top {
     height: 100% !important;
 }
+
+.p-heading {
```

Comment:
What does the `p-` stand for? Is it a way of sort of name-spacing CSS? A bit like `.l-` means `layout`?

Reply:
Exactly as you said! I've tried to make those classes unique to avoid classes collisions and weird selectors. In this example `p-` stands for `payment`

Reply:
p- stands for payment as tried to implement as custom as possible and reuse existing  css classes.